docs/dev/api-usage.rst

```diff
@@ -0,0 +1,93 @@
+.. role:: py(code)
+   :language: python
+
+API Usage Examples
+==================
+
+.. note::
+
+   These examples are written in Python version **3.6.3** and use `requests
+   <http://docs.python-requests.org/en/master/>`_ version **2.18.4**.
+
+Prerequisites
+#############
+
+All the examples assume you have a development version of the ListenBrainz
+server set up on :code:`localhost`. Remember to set :code:`DEBUG` to :py:`True`
+in the config. When in production, you can replace :code:`localhost` with
+:code:`api.listenbrainz.org` to use the real API. In order to use either
+one, you'll need a token. You can find it under :code:`ROOT/profile/`
+when signed in, with :code:`ROOT` being either :code:`localhost` or
+:code:`api.listenbrainz.org`.
+
+.. caution::
+
+   Use the token from the version you're testing with. You can't
```

Comment:
Ideally, we wouldn't want to encourage people to test the API by submitting test data to production. I think we should remove this.

Reply:
Okay, I'll rewrite this to say "You should use the token from the API you're using. In production, change the token to one from `listenbrainz.org.`" Is that good?